.circleci/config.yml

```
@@ -34,7 +34,7 @@ save-flow-typed-cache: &save-flow-typed-cache
     paths:
       - ./flow-typed
 
-npm-cache-key: &npm-cache-key node-v10-{{ checksum "yarn.lock" }}
+npm-cache-key: &npm-cache-key node-v10-{{ checksum "yarn.lock" }}-3
```

Why 3? Why should format this string properly, `v10` should say the version of the cache, not node.. So I would change it altogether to
```suggestion
npm-cache-key: &npm-cache-key node-cache-{{ checksum "yarn.lock" }}-v2
```